build.gradle.kts

```
@@ -291,11 +291,11 @@ val systemTestRuntimeOnly: Configuration by configurations.getting {
 }
 
 dependencies {
-    systemTestImplementation("com.hivemq:hivemq-testcontainer-junit5:${property("hivemq-testcontainer.version")}")
-    systemTestImplementation("org.testcontainers:testcontainers:${property("testcontainers.version")}")
-    systemTestImplementation("org.awaitility:awaitility:${property("awaitility.version")}")
-    systemTestImplementation("com.hivemq:hivemq-community-edition-embedded:${property("hivemq-community-edition-embedded.version")}")
-    systemTestImplementation("org.junit-pioneer:junit-pioneer:${property("junit-pioneer.version")}")
+    systemTestImplementation(libs.hivemq.testcontainer.junit)
```

```suggestion
    systemTestImplementation(libs.hivemq.testcontainer.junit5)
```
There are junit 5 and 4 versions, so the name should be distinct.